packages/dev/docs/src/Layout.js

```diff
@@ -148,6 +148,24 @@ function Page({children, currentPage, publicUrl, styles, scripts}) {
         <meta property="og:image" content={`https://${TLD}${heroImage}`} />
         <meta property="og:description" content={description} />
         <meta property="og:locale" content="en_US" />
+        <script
+          type="application/ld+json"
+          dangerouslySetInnerHTML={{__html: `
+            {
+              "@context": "http://schema.org",
+              "@type": "Article",
+              "author": "Adobe Inc",
+              "headline": "${currentPage.title}",
```

Comment:
may wanna use JSON stringify ensure these get escaped properly in case there's quotes in any of them. below ones too

```suggestion
              "headline": ${JSON.stringify(currentPage.title)},
```

Reply:
or, JSON stringify the whole thing to minify the whitespace